darksidemanas.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "darksidemanas"
+project: "medical sos emergeny recordig and monitoring system"
+---
+
+# Project Name
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY 
+3.PCB TOH CONNECT DIFFRENT MEDICAL PROBES WITH PI 
+4.TESTING WITH DIFFRENT MEDICAL SENSORS 
+5.DESINGING A METAL ENCLOSER TO PROTECT PI AND OTHER PCB .
```

Comment:
```suggestion
5.DESIGNING A METAL ENCLOSURE TO PROTECT PI AND OTHER PCB .
```

Reply:
how do you plan on building this metal enclosure?

Reply:
cool! and do you already have access to an appropriate 3d printer or other resources you need to design the case?

Reply:
yup in my community I have access to 3d printer ,well I m not sure about it is capable of printing metals or not but I am sure about ABS .

Reply:
sounds good, thanks!